spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require './lib/takeaway'
```

Comment:
Good practice to create a "describe" block specifying the name of the method being tested for each method.